txmix/client.py

```diff
@@ -60,10 +62,33 @@ def send(self, route, message):
         self.transport.send(first_hop_addr, raw_sphinx_packet)
 
 
+@implementer(IRouteFactory)
+@attr.s
+class RandomRouteFactory(object):
+    """
+    I create random routes.
+    """
+    params = attr.ib(validator=attr.validators.instance_of(SphinxParams))
+    pki = attr.ib(validator=attr.validators.provides(IMixPKI))
+    rand_reader = attr.ib(validator=attr.validators.provides(IReader))
+
```

Comment:
its not clear why this is called rand_reader (not e.g. just "reader") nor what it does to me; maybe a comment?